### PR TITLE
Update glossary.md

### PR DESCRIPTION
**Pull request by yvonneanne**
* remove platform
* make a few definitions more precise

### CODE REVIEW

File: docs/references/glossary.md [hunk 1]
```diff
@@ -220,17 +221,18 @@ correspond to one **XDR**.
...
 
 #### data center
 
 A **data center** (DC) is a physical site that hosts
 [nodes](#node) which contribute to the [Internet
 Computer](#internet-computer-ic). It includes the hardware and
-software infrastructure required for node deployment. A DC is identified
-on the Internet Computer by a unique identifier.
+software infrastructure required for node deployment. 
+Data centers are node are selected and vetted by the [NNS](#NNS).
```
Comment (jessiemongeon1) [inline docs/references/glossary.md:235]:
```suggestion
Data centers are nodes that are selected and vetted by the [NNS](#NNS).
```

Comment (yvonneanne) [inline docs/references/glossary.md:235]: Are there other aspects of data centers we want to highlight, @dprats ?

File: docs/references/glossary.md [hunk 2]
```diff
@@ -484,7 +488,8 @@ parameter.
 
 #### node
 
-A **node** is a physical or virtual network endpoint that hosts all the
+A **node** is a physical hardware devices run by independent
```
Comment (jessiemongeon1) [inline docs/references/glossary.md:491]:
```suggestion
A **node** is a physical hardware device run by independent
```

**Review by jessiemongeon1 — APPROVED** [review]

File: docs/references/glossary.md [hunk 3]
```diff
@@ -494,20 +499,8 @@ Computer](#internet-computer-ic).
...
-process consists essentially in deriving an IC principal ID from the key
-stored on the HSM, and persisting that ID with the NNS.) NPs hand
-registered HSMs over to legal persons, who now gain the authority to
-physically “operate nodes” (aka install
-[replicas](#replica)). The caveat is that, as opposed to
-"regular" principals, where a great deal of care goes into making sure
-that one principal ID corresponds to only one person, HSMs can routinely
-exchange hands, hence many persons can act as the same NO principal at
-different times.
+[IC](#internet-computer-ic-ic), delegated by
```
Comment (yvonneanne) [inline docs/references/glossary.md:502]:
```suggestion
[IC](#internet-computer-ic-ic), by delegation of the
```

**Pull request merged**